includes/admin/views/html-admin-page-status.php

```diff
@@ -35,7 +35,11 @@
 				WC_Admin_Status::status_logs();
 			break;
 			default :
-				WC_Admin_Status::status_report();
+				if ( array_key_exists( $current_tab, $tabs ) && has_action( 'woocommerce_admin_status_' . $current_tab ) ) {
```

Comment:
@allendav can you change it?
We can change to `'woocommerce_admin_status_content_' . $current_tab`